doc/modules/clustering.rst

```diff
@@ -1388,3 +1388,29 @@ Drawbacks
   concepts of clusters, such as density based clusters like those obtained
   through DBSCAN.
 
+.. _contingency_matrix:
+
+Contingency Matrix
+----------------------
+
+Contingency matrix is a format describing the relationship between discrete-valued random variables.
+It shows the multivariate frequency distribution of the variables.
+
+Let the true label set be :math:`T`, and the predicted label set be :math:`P`. In a contingency matrix :math:`M`, entry :math:`M(i, j)` 
```

Comment:
Can you maybe add a sentence that compares this to the ``confusion_matrix``? and maybe also add a link to that. The difference in our implementation is that we assume the label sets to be the same for ``confusion_matrix`` but not necessarily for ``contigency matrix``.